src/include/migraphx/apply_alpha_beta.hpp

```diff
@@ -0,0 +1,77 @@
+#ifndef MIGRAPHX_GUARD_MIGRAPHX_APPLY_ALPHA_BETA_HPP
+#define MIGRAPHX_GUARD_MIGRAPHX_APPLY_ALPHA_BETA_HPP
+
+#include "migraphx/errors.hpp"
+#include <migraphx/instruction.hpp>
+#include <migraphx/instruction_ref.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/common.hpp>
+#include <migraphx/module.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+template <typename T = float>
+instruction_ref insert_apply_alpha_beta(module& m,
+                                        instruction_ref pos,
+                                        const std::vector<instruction_ref>& args,
+                                        const std::string& op_name,
+                                        T alpha = 1.0f,
+                                        T beta  = 0.0f)
+{
+    if(!(op_name == "dot" or op_name == "quant_dot"))
```

Comment:
I dont think we should check the op_name is "dot" or "quant_dot". This function can be used for convolutions as well.

Reply:
removed the Check.